### PR TITLE
category options & google map validate address

### PR DESCRIPTION
**Pull request by JINGYANGJY**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,15 +4,20 @@
   "private": true,
   "dependencies": {
     "@ant-design/icons": "^4.3.0",
+    "@react-google-maps/api": "^2.0.2",
```
Comment (Alicewillbe) [inline package.json:7]: Unused dependency.
This lib is unmaintained and seems like not used in other files.

File: package.json [hunk 2]
```diff
@@ -4,15 +4,20 @@
...
-    "antd": "^4.8.6",
+    "antd": "^4.9.4",
+    "google-map-react": "^2.1.9",
     "history": "^5.0.0",
     "md5": "^2.3.0",
+    "prop-types": "^15.7.2",
     "react": "^17.0.1",
     "react-cookie": "^4.0.3",
     "react-dom": "^17.0.1",
+    "react-google-maps": "^9.4.5",
```
Comment (Alicewillbe) [inline package.json:19]: Unused.

File: package.json [hunk 3]
```diff
@@ -4,15 +4,20 @@
...
+    "@react-google-maps/api": "^2.0.2",
     "@testing-library/jest-dom": "^5.11.6",
     "@testing-library/react": "^11.2.2",
     "@testing-library/user-event": "^12.2.2",
-    "antd": "^4.8.6",
+    "antd": "^4.9.4",
+    "google-map-react": "^2.1.9",
     "history": "^5.0.0",
     "md5": "^2.3.0",
+    "prop-types": "^15.7.2",
```
Comment (Alicewillbe) [inline package.json:15]: What is this for?

File: package.json (hunk 1 above)
Comment (JINGYANGJY) [reply package.json:7]: sorry about forgetting to remove it from the package.json

File: package.json (hunk 3 above)
Comment (JINGYANGJY) [reply package.json]: unused, for trying during implementation

**Pull request closed without merging**